src/api/players/client/canUseRegisteredKey.lua

```diff
@@ -0,0 +1,16 @@
+---canUseRegisterKey
+---@param Bool boolean
+---@param ButtonIndex string
+function _NCS:canUseRegisteredKey(Bool, ButtonIndex)
+    if (type(Bool) == 'boolean') then
+        for _,v in pairs(registeredKeysTable) do
+            if (ButtonIndex) then
+                if (v.key == string.upper(ButtonIndex)) then
+                    v.canUse = Bool
+                end
+            else
+                v.canUse = Bool
+            end
+        end
+    end
+end
```

Comment:
Please consider using camelCase for variable names.

![image](https://user-images.githubusercontent.com/42814853/170708746-b85fdf25-2103-4df7-a6d0-2668e30eb2b0.png)